theorems/T000083.md

```diff
@@ -4,13 +4,14 @@ if:
   and:
   - P000043: true
   - P000052: false
-  - P100052: false
 then:
   P000058: false
 refs:
 - doi: 10.1007/978-1-4612-6290-9
   name: Counterexamples in Topology
 ---
 
-If any point of $X$ is not isolated and $X$ is arc connected at that point, then $X$ contains the injective image of an arc.
+By non-discreteness choose a non-isolated point of \(X\).
+It has an arc connected neighborhood with two distinct points,
```

Comment:
"with at least two distinct points" ?

Reply:
This and your other suggestion sound reasonable - feel free to make a PR to that effect.